react/features/recording/components/Recording/StartRecordingDialogContent.js

```diff
@@ -107,10 +155,47 @@ class StartRecordingDialogContent extends Component<Props> {
      * @returns {React$Component}
      */
     _renderNoIntegrationsContent() {
+
+        if (!this.props.fileRecordingsServiceEnabled
+            && this.props.integrationsEnabled) {
+            return null;
```

Comment:
I'm a little confused about what this method is doing versus. This method specifically is for jitsi's file recording service? But it also handles displaying integrations?

Reply:
Well, these are the conditions when to show/hide the non-integration part.
Show it when
- fileRecordingsServiceEnabled is enabled
- or show it when there are no integrations
I will reverse it then and will add a comment.

Reply:
Done.